src/styles/blocks/card.scss

```diff
@@ -0,0 +1,156 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  place-content: center;
+  place-items: center;
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
```

Comment:
The border property should be defined by the variables for border-width and border-style according to the code style rules. This will ensure consistency and ease of maintenance.